polyglot.js

```diff
@@ -362,18 +380,86 @@ class PolyGlot {
         }
         sheet._polyglotEditor = true;
     }
-    renderJournalSheet(journalSheet) {
+
+    renderJournalSheet(journalSheet,html) {
         this._addPolyglotEditor(journalSheet);
-        if (journalSheet.entity.owner) return;
-        if (game.user.isGM && !game.settings.get("polyglot", "runifyGM")) return;
+        if (journalSheet.entity.owner || game.user.isGM) {
+            let runes = false;
+            let texts = [];
+            let openBtn = $(`<a class="polyglot-button" title="Toggle Runes"><i class="fas fa-unlink"></i> Runes</a>`);
```

Comment:
use `const for `texts` and `openBtn`.
Also, rename `openBtn` into `toggleButton` since it's a toggle, and it's not actually opening anything.